core/rend/dx11/dx11_renderer.cpp

```diff
@@ -1294,6 +1300,93 @@ void DX11Renderer::writeFramebufferToVRAM()
 	WriteFramebuffer<2, 1, 0, 3>(width, height, (u8 *)tmp_buf.data(), texAddress, pvrrc.fb_W_CTRL, linestride, xClip, yClip);
 }
 
+#ifdef VIDEO_ROUTING
+void DX11Renderer::RenderVideoRouting()
+{
+	if (config::VideoRouting)
+	{
+		extern void os_VideoRoutingPublishFrameTexture(ID3D11Texture2D* pTexture);
+		
+		ID3D11RenderTargetView* pRenderTargetView = theDX11Context.getRenderTarget().get();
+
+		// Backbuffer texture would be different after reszing, fetching new address everytime
+		ID3D11Resource* pResource = nullptr;
+		pRenderTargetView->GetResource(&pResource);
+		ID3D11Texture2D* pTexture = nullptr;
+		pResource->QueryInterface(__uuidof(ID3D11Texture2D), (void**)&pTexture);
+
+		static ID3D11Texture2D* backBufferTexture = nullptr;
+		if (backBufferTexture != pTexture)
+		{
+			backBufferTexture = pTexture;
```

Comment:
Same reference counting issue here: you keep a (static!) reference to `pTexture` via `backbufferTexture` after calling Release.
Do you really need a static variable here?
If not move the move the `pTexture->Release()` to the end of the variable scope.
If yes you need to call `Release()` on it when assigning a new reference.